src/TouristInfoCards.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+
+const TouristInfoCards = () => {
+return (
+     <ul className="d-flex justify-content-around">
```

Comment:
Consider implementing a separate tourist card component too here - you're duplicating the div with className "card".